src/app/views/record/record.html

```diff
@@ -42,7 +67,9 @@ <h3>{{ $ctrl.record.entity }} {{ $ctrl.getIdentifier($ctrl.record) }}</h3>
     <table flex>
       <tbody>
         <tr ng-repeat="(key, value) in $ctrl.auxiliaryRecordDetails(0) track by key">
-          <th>{{ key }}:</th>
+          <th ng-class="{'error': !value.href && $ctrl.record.entity === 'Sample_Photo'}">
```

Comment:
this logic will make any attribute that isn't have a href contain the error class. This is not what we want. Better to just explicitly check `key==='imageProcessingErrors'`

Reply:
this also needs to work w/ event photos